cdc/log.cc

```diff
@@ -68,6 +69,70 @@ namespace cdc {
 static schema_ptr create_log_schema(const schema&, std::optional<utils::UUID> = {});
 }
 
+static constexpr auto cdc_group_name = "cdc";
+
+void cdc::stats::parts_touched_stats::register_metrics(seastar::metrics::metric_groups& metrics, sstring suffix) {
+    namespace sm = seastar::metrics;
+    auto register_part = [&] (part_type part, sm::label_instance part_instance) {
+        metrics.add_group(cdc_group_name, {
+                sm::make_total_operations("components_touched_" + suffix, count[(size_t)part],
+                        sm::description("number of times a mutation part type have been touched during cdc mutation augmentation"),
```

Comment:
the description is not clear to me, if that's for internal use only, it's fine, if you think users should look at it, it should be clearer.

Reply:
I'm not a big fan of 'touched'. What does it mean? If an operation changes static row and two clustering rows how many touches are there? I would expect to count one static row operation and one clustering row operation.

Reply:
@haaawk Your understanding is correct, this counter will be incremented as you described.

I'm not sure if `operations_performed` is a good name by itself here. There will be many `operations_performed` metrics, differing by the label that determines which part of the mutation was "touched". If we sum them all up, a number greater then the actual total number of operations performed will appear.

Perhaps we should abandon labels and give different name for those counters, e.g. `operations_on_static_row_performed`, `operations_on_clustering_row_performed`?

Reply:
It might be a solution. Labels may not be a good fit here.